lib/maplibre-compose/src/commonMain/kotlin/dev/sargunv/maplibrecompose/compose/CameraState.kt

```diff
@@ -42,8 +44,8 @@ public class CameraState internal constructor(firstPosition: CameraPosition) {
       positionState.value = value
     }
 
-  public val isInitialized: Boolean
-    get() = map != null
```

Comment:
removing this as it wasn't state and so might be confusing

if someone needs it someday, reimplement with state